TrainingMaquetado/scss/tech.scss

```diff
@@ -0,0 +1,41 @@
+.tech {
+  position: relative;
+
+  &::before {
+    content: '';
+    width: 100%;
+    height: 100%;
+    border: 1px solid #BED23C;
+    position: absolute;
+    bottom: 10px;
+    right: 10px;
+    border-radius: 5px;
+    z-index: -1;
+  }
+
+  align-items: center;
+  background: #BED23C;
+  border-radius: 4px;
+  display: inline-flex;
+  flex-direction: column;
+  font-size: 16px;
+  margin: 10px 50px 50px 20px;
```

Comment:
Avoid top and left. In almost every scenario you can solve this by using bottom and right in some other element or even padding in your container